src/types/course/enums/course.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum CourseAutocompleteOptionsEnum {
+  Categories = 'Categories',
```

Comment:
`Categories` already exist in `ResourcesTabsEnum`
Maybe pass there `Subjects` and reuse enum and rename smth like `ResourcesEnum`

Reply:
I don`t think ResourcesEnum is a good naming, because subjects is not a resource, To my mind, this enum looks great

Reply:
> I don`t think ResourcesEnum is a good naming, because subjects is not a resource, To my mind, this enum looks great

just we can reuse existing enum and not to create another one with properties and give better naming for both

Reply:
> I didn`t see any enum with 'Categories' and 'Subject' properties

'Categories' there is in `ResourcesTabsEnum`.
In our case 'categories'. with first lowercase letter.
so, it make sense to create another enum with your naming